src/vs/workbench/parts/extensions/browser/extensionsList.ts

```diff
@@ -176,6 +177,20 @@ export class Renderer implements IPagedRenderer<IExtension, ITemplateData> {
 		data.installCount.style.display = '';
 		data.ratings.style.display = '';
 		data.extension = extension;
+
+		this.getLanguagePackLanguageNameIfPresent(extension).then(name => {
+			if (name) { data.description.textContent = name; }
+		});
+	}
+
+	private getLanguagePackLanguageNameIfPresent(extension: IExtension): TPromise<string | undefined> {
+		return extension.getManifest().then(manifest => {
```

Comment:
let's move this piece of code inside `renderElement`. Since this code does not get used from more than 1 location, there is not much benefit of making a dedicated function for it.